package.json

```diff
@@ -4,7 +4,11 @@
   "description": "Webpack Encore is a simpler way to integrate Webpack into your application",
   "main": "index.js",
   "scripts": {
-    "test": "mocha --reporter spec test --recursive",
+    "test": "run-s test:*",
```

Comment:
Why `run-s` vs `yarn test:normal && yarn:test-persistent-cache-clear && `, etc?

Reply:
Mainly because it is just simpler (to read) and easier to extend with new scripts in the same namespace in the future. And partly personal preference. 
If it needs to be adjusted, just say the word!